angr/engines/concrete.py

```diff
@@ -37,14 +37,15 @@ def __init__(self, project):
 
         self.segment_registers_already_init = False
 
-    def __check(self, state, *args, **kwargs):
+    def __check(self, _, *args, **kwargs):  #pylint:disable=unused-argument
         return True
```

Comment:
Oh, important note, you can actually just remove the check method now. It got made extra private in some classes because it's not part of any interface anymore, so where it exists is strictly just for code organization.